src/mlpack/core/data/load_csv.cpp

```diff
@@ -25,15 +25,30 @@ LoadCSV::LoadCSV(const std::string& file) :
   CheckOpen();
 
   // Set rules.
-  if (extension == "csv" || extension == "txt")
+  if (extension == "csv")
```

Comment:
It might be nice to make these changes for the `txt` files too; I could have something like this:

```
one "long two" three
```

(Basically, I consider a .txt file a .csv that's space-delimited.)  Let me know what you think---maybe there is some complexity there I've overlooked.

Reply:
@rcurtin  I will add for txt files as well.